package.json

```diff
@@ -93,9 +101,9 @@
     "eslint-plugin-jest-playwright": "^0.2.1",
     "eslint-plugin-prettier": "^3.1.2",
     "husky": "^3.1.0",
-    "jest": "^26.4.2",
+    "jest": "^27.0.0",
     "jest-image-snapshot": "^4.2.0",
-    "jest-playwright-preset": "^1.4.1",
+    "jest-playwright-preset": "^1.7.0",
```

Comment:
updating Jest dependencies seems to have somehow causes `jest-image-snapshot` to produce slightly different font rendering, requiring an update to one of our visual snapshots.

I verified locally that both develop and this branch look identical by alternating between tabs in Chrome, and we can also verify this using Vercel's preview.